prog/remove_boot_image.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class Prog::RemoveBootImage < Prog::Base
+  subject_is :boot_image
+
+  label def start
+    boot_image.update(activated_at: nil)
+    hop_wait_volumes
+  end
+
+  label def wait_volumes
+    nap 30 if boot_image.vm_storage_volumes.count > 0
+    hop_remove
+  end
+
+  label def remove
+    # YYY: version will be enforced in future.
+    image_path =
+      boot_image.version ?
+          "/var/storage/images/#{boot_image.name}-#{boot_image.version}.raw" :
+          "/var/storage/images/#{boot_image.name}.raw"
```

Comment:
The `DownloadBootImage` also has the same path check. Should we consider moving it to the `BootImage` model as a helper, like `boot_image.file_path`, `boot_image.image_path`?